PWGGA/GammaConvBase/AliConversionMesonCuts.h

```diff
@@ -311,7 +311,7 @@ class AliConversionMesonCuts : public AliAnalysisCuts {
     Bool_t      fDoJetPtMixing;                 ///< flag to enbale mixing by jet pt bins
     Bool_t      fDoSphericityMixing;            ///< flag to enable Sphericitymixing for meson bg estimation
     Bool_t      fUseTrackMultiplicityForBG;     ///< flag to use track multiplicity for meson bg estimation (else V0 mult)
-    Bool_t      fDoGammaSwappForBg;             ///< flag to use cluster swapping for background estimation
+    Int_t       fDoGammaSwappForBg;             ///< flag to use cluster swapping for background estimation
```

Comment:
please increment the classdef

Reply:
Thanks for reminding me!
Updated both class defs now.

Reply:
this request was only for conversionmesoncuts

Reply:
for your user task it is not needed as you just add a transient member variable (//!)

Reply:
So when do you have to increment it?

Reply:
Whenever you add/remove a non-transient member variable

Reply:
ClassDef increment is now missing for MesonCuts - why did you touch AliConversionCuts? There are no real changes, but a ClassDef increment - please remove that from your PR

Reply:
Learned something new again. :)
Reverted the change for my task.